pkg/util/v1alpha2/metricscollector/metricscollector.go

```diff
@@ -46,18 +45,15 @@ func (d *MetricsCollector) CollectObservationLog(tId string, jobKind string, met
 	if len(pl.Items) == 0 {
 		return nil, fmt.Errorf("No Pods are found in Trial %v", tId)
 	}
-	logopt := apiv1.PodLogOptions{Container: "tensorflow", Timestamps: true, Follow: true}
```

Comment:
I think we should keep the file unmodified since we have pkg/util/v1alpha2/sidecarmetricscollector